### ISSUE TITLE
BRO you can add Ibernation mode without lost mod ps4? 

### ISSUE THREAD

**Issue by DarkGod1** [sleirsgoevy/ps4jb#84]
another developer already make it for dont lost mod 7.55 when go in sleep mode, maybe you can implement it on your mod? it can be great... because i want game on hamachi but in the version goldhen hamachi dont work.... or you can send a version of hamachi compatible for goldhen?

**Comment by sleirsgoevy** [comment]
HEN support is planned for ps4-hamachi. Post an issue there and give more details on how exactly it does not work

**Comment by sleirsgoevy** [comment]
Just tested the latest build with GoldHEN. Seems to work just fine.

**Comment by DarkGod1** [comment]
Yes i successful at last.. but only with this trick: i install your version and install hamachi .. start and aftet i changr with goldhen and it works but i dont try in game for.now..... you thinks is possible make a playstore bypass some day? For real game online  :)
________________________________
Da: sleirsgoevy ***@***.***>
Inviato: giovedì 25 marzo 2021 09:32
A: sleirsgoevy/ps4jb ***@***.***>
Cc: DarkGod1 ***@***.***>; Author ***@***.***>
Oggetto: Re: [sleirsgoevy/ps4jb] BRO you can add Ibernation mode without lost mod ps4? (#84)


Just tested the latest build with GoldHEN. Seems to work just fine.

**Comment by DarkGod1** [comment]
o bro you thinks is impossible modify ps share play function and apply it in lan with hamachi?? Is genial in this way all people can share  the second controller for game together
________________________________
Da: sleirsgoevy ***@***.***>
Inviato: giovedì 25 marzo 2021 09:32
A: sleirsgoevy/ps4jb ***@***.***>
Cc: DarkGod1 ***@***.***>; Author ***@***.***>
Oggetto: Re: [sleirsgoevy/ps4jb] BRO you can add Ibernation mode without lost mod ps4? (#84)


Just tested the latest build with GoldHEN. Seems to work just fine.
